frontend/src/metabase/static-viz/components/XYChart/types.ts

```diff
@@ -15,7 +15,12 @@ export type YAxisType = "linear" | "pow" | "log";
 
 export type YAxisPosition = "left" | "right";
 
-export type VisualizationType = "line" | "area" | "bar";
+export type VisualizationType =
+  | "line"
+  | "area"
+  | "bar"
+  | "waterfall"
+  | "waterfall-total";
```

Comment:
I feel a bit uncomfortable, adding `waterfall-total` here, but it's quick and easy way to make `waterfall` chart with `showTotal: true` renders data point value correctly, without altering various components' APIs too much.